docs/content/intro/community-roles.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Participating in the Athens Community"
+date: 2018-02-11T15:57:56-05:00
+---
+
+Absolutely everyone is welcome to join our community at any time! We
+are a friendly and inclusive group and we'd love to have you. We have three roles in the Athens community:
+
+- Community member
+- Contributor
+- Maintainer (we also call this core maintainer sometimes. They're the same thing)
+
+Read on to find out more!
+
+# Community Members
+
+Community members are folks who decide they want to get involved with our
+community. Absolutely anyone can do that whenever they want. If you want
+to get involved, that doesn't mean you have to continue being involved, but
+we hope our community is welcoming and the work is interesting enough to 
+convince you to say :)
+
+We'll provide all the support we can possibly provide to help you contribute
+in any way you'd like. If you're considering joining us, here are some ideas
+for how you can get involved:
+
+- Comment on an [issue](https://github.com/gomods/athens/issues) that you're
+interested in
+- Submit a [pull request](https://github.com/gomods/athens/pulls) (PR) to fix 
+an issue, or to improve something that doesn't have an issue
+- Review a PR that you're interested in
+- Join us at a weekly [development meeting](https://docs.google.com/document/d/1xpvgmR1Fq4iy1j975Tb4H_XjeXUQUOAvn0FximUzvIk/edit#)
+(or more than one!)
+- Come chat with us in the [gophers slack](https://invite.slack.golangbridge.org/) in the `#athens` channel
+- ... and anything else that's appropriate for you!
+
+# Contributors
+
+As you participate in the community more and more, you'll have the opportunity 
+to become a contributor. Here's what being a contributor means, and what you
+should do to become one.
+
+## What Being a Contributor Means
+
+As a contributor, you'll have issues assigned to you and you'll be requested 
+to review pull requests (PRs) via the 
+[Github pull request review system](https://help.github.com/articles/about-pull-request-reviews/). 
+We rely heavily on Github PR reviews, so if you review a PR as a contributor, you can decide 
+when that PR is ready to be merged.
+
+## How to Become a Contributor
+
+To become a contributor, the core maintainers of the project would like to see you:
+
+- Attend our development meetings regularly<sup>1</sup>
+- Comment on issues with your experiences and opinions
+- Add your comments and reviews on pull requests (anyone can do this as a community member)
+- Contribute PRs to fix issues
+- Open issues as you find them
```

Comment:
It would help if we could say "after doing this for X amount of time, email X or open an issue or whatever and ask to be made a contributor". 

For example, have I been on the project long enough to be considered a contributor?

Reply:
i dont think X is right thing to do. might lead to too many small PR just to hit the X. 
I would Go with making your work visible. sticking around...i'm afraid that X will make it a race. 
i see it just like a regular job. you don't get promoted because you completed 101 PRs, but because you proved yourself. 
not sure if this makes sense :-)

Reply:
Bad wording on my part. I wasn't suggesting a hard number. But calling out that what we do now is so vague that it's really hard to tell when someone is ready to move to the next level and how to do so. Most people are too intimidated to say "I think I should be a member of the athens org/made a maintainer" etc and then hunt down the right person to say that to! 

Like I've been contributing but have no idea if I've passed the magical bar to be promoted to a contributor, or how to ask for that without looking like an entitled jerk. 😉 If I'm confused, other people are too.

Reply:
Sound good. Some concrete goals people can try to achieve. Gamifying the whole procedure to become an owner. 
You mentioned you wrote guidelines for kubernetes. Do you have, by any chance, an example of well-formed 'ladder progression' here?

Reply:
Agree that we shouldn't put the onus on the community member to "request" to be a contributor, or a contributor to become a member. 

Wondering if we could have:

- contributors+maintainers meet regularly to review community contributions and nominate folks for contributor
- maintainers meet regularly to review contributors and nominate folks for maintainer

Both would be based on the criteria above. Also, totally agree we need to have some more explicit expectations - I just don't know what they should be without making it really intense. What do folks think? Any examples that people have seen in other projects (I looked at a few Kubernetes projects, didn't find any good examples)?

Thoughts?

Reply:
👍 for regular meetings

Reply:
Here are k8s requirements for becoming the equivalent of our contributor (their member):

https://github.com/kubernetes/community/blob/master/community-membership.md#requirements

It relies on someone asking, but the list of what they should have done is mostly clear. Since it requires 2 sponsors, what many sponsor-eligible people do is keep an eye out for people who fit the criteria and then offer to sponsor them and encourage them to go for it.

Reply:
I like rolling it up into our regular meetings, or somehow just doing a regular look around to check for people who are ready to be made contributors/maintainers.